addons/mail/static/src/models/message_action_view.js

```diff
@@ -0,0 +1,60 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+import { clear, insertAndReplace, replace } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'MessageActionView',
+    identifyingFields: ['messageAction'],
+    recordMethods: {
+        /**
+         * @private
+         * @param {MouseEvent} ev
+         */
+        onClickReaction(ev) {
+            this.messageActionList.message.addReaction(ev.currentTarget.dataset.unicode);
+            this.update({ reactionPopoverView: clear() });
+        },
+        /**
+         * @private
+         */
+        openReactionPopver() {
+            if (!this.reactionPopoverView) {
+                this.update({ reactionPopoverView: insertAndReplace() });
+            } else {
+                this.update({ reactionPopoverView: clear() });
+            }
+        },
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeMessageActionList() {
+            return replace(this.messageAction.messageActionListOwner);
+        },
```

Comment:
this could just be a related (but do we actually need this field instead of just following relations?)